config/projects/angrychef.rb

```diff
@@ -47,6 +47,17 @@
 compress :dmg
 
 package :msi do
+  install_path = Pathname.new(install_dir)
+
+  # Find path in which the Chef gem is installed
+  search_pattern = "#{install_path}/**/gems/chef-[0-9]*-mingw32/ext/win32-eventlog/chef-log.dll"
```

Comment:
No better or worse than what it currently does.  The paths are still going to contain mingw32/ext/win32-eventlog unless we rename it.  Whether or not the dll we compile will actually work...  *shrug*.  We already use an old deprecated API IIRC - so we'll probably need to take a look at it separately when dealing with Nano.